core/src/main/scala/kafka/raft/KafkaMetadataLog.scala

```diff
@@ -22,13 +22,12 @@ import kafka.log.{AppendOrigin, Log}
 import kafka.server.{FetchHighWatermark, FetchLogEnd}
 import org.apache.kafka.common.KafkaException
 import org.apache.kafka.common.record.{MemoryRecords, Records}
-import org.apache.kafka.common.utils.Time
 import org.apache.kafka.raft
 import org.apache.kafka.raft.{LogAppendInfo, ReplicatedLog}
 
 import scala.compat.java8.OptionConverters._
 
-class KafkaMetadataLog(time: Time, log: Log, maxFetchSizeInBytes: Int = 1024 * 1024) extends ReplicatedLog {
+class KafkaMetadataLog(log: Log, maxFetchSizeInBytes: Int = 1024 * 1024) extends ReplicatedLog {
```

Comment:
Haha I was wondering if you kept `time` for some future works so I did not remove it :P

Reply:
Haha, I was wondering if I did the same. But it's always cheap to add it back if I ever remember the reason.